src/main/java/dev/joopie/jambot/repository/SpotifyToYoutubeRepository.java

```diff
@@ -0,0 +1,16 @@
+package dev.joopie.jambot.repository;
+
+import dev.joopie.jambot.models.SpotifyToYoutube;
+
+import javax.xml.bind.ValidationException;
+import java.util.List;
+
+public interface SpotifyToYoutubeRepository {
```

Comment:
With this method we can actually expose methods that we want from the BaseRepository without much hassle.